modules/logging/encoders.go

```diff
@@ -266,6 +275,8 @@ func (lec *LogEncoderConfig) ZapcoreEncoderConfig() zapcore.EncoderConfig {
 		durFormatter = zapcore.NanosDurationEncoder
 	case "string":
 		durFormatter = zapcore.StringDurationEncoder
+	case "ms":
```

Comment:
The options are kinda misaligned I think, nano and ms aren't the same, should be nano and milli for example. But we could support all of them. So ns/nano, s/second, ms/milli etc

Reply:
Better?

Reply:
Yep, thanks!

Thinking we should probably add docblock comments to the rest of the fields too, but this is good enough for now.